version 1.1.1 (#425)
sentinel-hub/eo-learn: CHANGELOG.md

## [Version 1.1.1] - 2022-06-14

> - Fixed a bug in `eolearn.io.ImportFromTiffTask` where a bounding box from an image wasn't written into an EOPatch.

Reviewer: ```suggestion
- Fixed a bug in `eolearn.io.ImportFromTiffTask` where a bounding box from the image wasn't added to the EOPatch.
```
